### PR TITLE
precommit: skip hooks with bot & ruff

### PR DESCRIPTION
**Pull request by Borda**
# What does this PR do?

This shall allow enabling bot: https://github.com/pre-commit-ci/issues/issues/7#issuecomment-775514948

## Submit checklist

- [x] My PR is focused - addressing only one thing at the time
- [x] I wrote new tests \[for a bug or new feature\]
- [ ] I manually QAed this PR in my local environment
- [ ] I added screenshots or screencasts featuring all UI & UX changes introduced by the PR

### Additional

- I added screenshots featuring related Figma designs and links to the corresponding Figma nodes
- My implementation follows Figma design as close as possible in terms of colors, sizes, margins, proportions, and
  positions

### CODE REVIEW

**Comment by Borda** [comment]
@yurijmikhalevich any idea why the `eslint` fails inside pre-commit but passes for the stand-alone call?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -54,8 +47,16 @@ repos:
...
-        name: PEP8
+      - id: ruff
+        args: ["--fix"]
+
+  - repo: local
+    hooks:
+      - id: ui_lint
+        name: ui_lint
+        description: Lint and fix (when possible) UI
+        entry: bash -c 'yarn lint:fix'
```
Comment (yurijmikhalevich) [inline .pre-commit-config.yaml:61]: `yarn lint:fix`won't error when `yarn lint` errors

pre-commit will fail if something modifies the files, right?

**Review by yurijmikhalevich — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (yurijmikhalevich) [reply .pre-commit-config.yaml:61]: if not, it can make sense to switch to `yarn lint` there

Comment (Borda) [reply .pre-commit-config.yaml:61]: > pre-commit will fail if something modifies the files, right?

correct

**Pull request merged**